frontend/test/metabase/scenarios/question/custom_column.cy.spec.js

```diff
@@ -55,4 +57,25 @@ describe("scenarios > question > custom columns", () => {
     firstCell("contain", 1);
     firstCell("not.contain", 14);
   });
+
+  it("should allow the use of custom column formulas", () => {
+    const customFormula = "1 + 1";
+    const columnName = "Simple math";
+
+    // go straight to "orders" in custom questions
+    cy.visit("/question/new?database=1&table=2&mode=notebook");
+    cy.get(".Icon-add_data").click();
+
+    popover().within(() => {
+      _typeUsingGet("[contenteditable='true']", customFormula);
+      _typeUsingPlaceholder("Something nice and descriptive", columnName);
+
+      cy.findByText("Done").click();
+    });
+
+    cy.findByText("Visualize").click();
+    // wait for visualization to render (Note: anti-pattern)
+    cy.wait(2000);
```

Comment:
We should definitely avoid `wait`s as much as we can. 

This is probably fine for this one, but we should either wait on `/api/dataset` (example below) or do a `cy.contains` on an element on the results page.

```
      cy.route("POST", "/api/dataset").as("dataset");
      // ... click done
      cy.wait("@dataset");
```